test/boost/repair_test.cc

```diff
@@ -186,7 +186,7 @@ SEASTAR_TEST_CASE(test_reader_with_different_strategies) {
             });
             auto read_all = [&](repair_reader::read_strategy strategy) -> future<std::vector<mutation_fragment>> {
                 auto reader = repair_reader(e.db(), cf, cf.schema(), make_reader_permit(e),
-                    random_range, remote_sharder, remote_shard, 0, strategy, gc_clock::now());
+                    random_range, remote_sharder, remote_shard, 0, strategy, gc_clock::now(), 128 * 1024);
```

Comment:
Isn't this tiny?

Reply:
This is a test, which runs on low amount of memory.